src/Features/Core/Portable/CodeRefactorings/CodeRefactoringService.cs

```diff
@@ -163,5 +176,107 @@ private IEnumerable<CodeRefactoringProvider> GetProviders(Document document)
 
             return null;
         }
+
+        private ImmutableArray<CodeRefactoringProvider> GetProjectRefactorings(Project project)
+        {
+            // TODO (https://github.com/dotnet/roslyn/issues/4932): Don't restrict refactorings in Interactive
+            return project.Solution.Workspace.Kind == WorkspaceKind.Interactive
+                ? ImmutableArray<CodeRefactoringProvider>.Empty
+                : _projectRefactoringsMap.GetValue(project.AnalyzerReferences, pId => new StrongBox<ImmutableArray<CodeRefactoringProvider>>(ComputeProjectRefactorings(project))).Value;
+        }
+
+        private ImmutableArray<CodeRefactoringProvider> ComputeProjectRefactorings(Project project)
+        {
+            var extensionManager = project.Solution.Workspace.Services.GetService<IExtensionManager>();
```

Comment:
i'm probably missing it... but i can't see how this variable is used.

Reply:
Removed